app.py

```diff
@@ -319,6 +320,7 @@ def get(self):
         values['hash'] = payload_id  # provided for backward compatibility
         values['size'] = size
         values['service'] = service
+        values['metadata'] = json.loads(self.metadata)
```

Comment:
It looks like `metadata` could be `None`

Reply:
It could be, but is that bad?

Reply:
nevermind. I see waht you're saying